Makefile

```diff
@@ -15,9 +15,10 @@
 ## program will usually tell you what to do.
 ##
 ## Once you have prepared your environment for running docker, you can
-## test the build by issuing 'make test', which at present is
-## synonymous with 'make html'. I.e., the website build is tested by
-## rendering all HTML content.
+## test the build by issuing 'USERID=0 GROUPID=0 make test', which at 
```

Comment:
Well, this is only true on MacOSX, and on Unix will actually cause the build to fail! Moreover, on Linux you have also have to set SUDO (to, in most cases, "sudo"), like so:

```sh
$ SUDO=sudo make html
```

or otherwise the invocation of Docker will typically fail. (But setting SUDO on MacOSX will most likely cause the build to fail.)

One wishes Docker behaved the same on all systems, but because systems like MacOSX have to run Docker within a VM, whereas it runs natively on Linux, the behaviors are different in sometimes subtle ways.